learing_journal/templates/layout.jinja2

```diff
@@ -19,8 +19,10 @@
     <link href="../../assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
```

Comment:
especially when dealing with files that are on relative paths.  This gets really tricky to do right in dynamic systems.  Always generate the "right" URL with `request.static_path`